dev-tools/generate_notice.py

```diff
@@ -82,16 +91,16 @@ def create_notice(filename, beat, copyright, licenses):
 
     cwd = os.getcwd()
     notice = os.path.join(cwd, "NOTICE")
-    licenses = []
+    license_globs = []
 
     for root, dirs, files in os.walk(args.vendor):
```

Comment:
I couldn't ignore dot directories for the walk but did ignore all dirs starting with a dot afterwards.